app/src/main/res/values/strings.xml

```diff
@@ -1713,4 +1713,8 @@
 
     <!-- Content description radio buttons with a link to more information -->
     <string name="radio_preference_info_content_description">Click for more details</string>
+
+    <!-- Content description for privacy content close button -->
+    <string name="privacy_content_close_button_content_description">close</string>
```

Comment:
Nit: even though the old string is "close", let's capitalize this "Close" for consistency 😄

Reply:
okay.

Reply:
@eliserichards also if it should be added to other strings.xml files

Reply:
Adding it here in the main file is all you need. Our translators will add it to the other files as localized strings become available :)

Reply:
> Adding it here in the main file is all you need. Our translators will add it to the other files as localized strings become available :)

ohhkk, Thanks for confirming☺

Reply:
@eliserichards I have capitalizes Close.